contracts/english_auction_rev1.1.scilla

```diff
@@ -147,7 +147,7 @@ contract EnglishAuctionOrderbook
 )
 
 (* Allowlist Contract Address *)
-(* Defaults to `initial_contract_owner` *)
+(* Defaults to `zero_address` *)
```

Comment:
@sam-zilliqa I think you would have to rebase from main since these audit changes are already there